virttest/qemu_devices/qcontainer.py

```diff
@@ -161,6 +164,10 @@ def get_qmp_cmds(qemu_binary, workaround_qemu_qmp_crash=False):
         self._probe_capabilities()
         self.__iothread_manager = None
         self.__iothread_supported_devices = set()
+        if images_offline_snapshots is None:
+            self.images_offline_snapshots = {}
+        else:
+            self.images_offline_snapshots = images_offline_snapshots
```

Comment:
And the method argument is not needed anymore.

Reply:
Thanks, have updated.